### PR TITLE
fix(curriculum): Convert blockquote elements to triple backtick syntax for Data Visualization 

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

**PR Purpose:** Converts `blockquote` elements to triple backtick syntax in the `Description` and `Instructions` sections of the `Data Visualization` challenges.

Related to issue #35991

### CODE REVIEW

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/access-the-json-data-from-an-api.english.md [hunk 1]
```diff
@@ -8,14 +8,25 @@ challengeType: 6
 <section id='description'>
 In the previous challenge, you saw how to get JSON data from the freeCodeCamp Cat Photo API.
 Now you'll take a closer look at the returned data to better understand the JSON format. Recall some notation in JavaScript:
-<blockquote>[ ] -> Square brackets represent an array<br>{ } -> Curly brackets represent an object<br>" " -> Double quotes represent a string. They are also used for key names in JSON</blockquote>
+
+```js
```
Comment (moT01) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/access-the-json-data-from-an-api.english.md]:
```suggestion
```
```

Comment (moT01) [reply curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/access-the-json-data-from-an-api.english.md]: I think it was decided to stick with the blockquotes on non-code - I'm fine with the unlabeled backticks - I'll leave it up to you guys - Other than this, it all looks good 🎉

**Comment by moT01** [comment]
There's one PR ahead of this - https://github.com/freeCodeCamp/freeCodeCamp/pull/35415

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/access-the-json-data-from-an-api.english.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/access-the-json-data-from-an-api.english.md]: Done.

**Comment by RandellDawson** [comment]
@moT01 Is this one good now?

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.english.md [hunk 2]
```diff
@@ -11,14 +11,37 @@ You can use a <code>forEach</code> method to loop through the data since the cat
...
+
 </section>
 
 ## Instructions
 <section id='instructions'>
 Add a <code>forEach</code> method to loop over the JSON data and create the HTML elements to display it.
 Here is some example JSON
-<blockquote>[</br>&nbsp;&nbsp;{</br>&nbsp;&nbsp;&nbsp;&nbsp;"id":0,</br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"imageLink":"https://s3.amazonaws.com/freecodecamp/funny-cat.jpg",</br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"altText":"A white cat wearing a green helmet shaped melon on its head. ",</br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"codeNames":[ "Juggernaut", "Mrs. Wallace", "Buttercup"</br>&nbsp;&nbsp;&nbsp;&nbsp;]</br>&nbsp;&nbsp;}</br>]</blockquote>
+
+```js
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.english.md]:
```suggestion
```json
```

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md [hunk 3]
```diff
@@ -11,7 +11,15 @@ The navigator will get the user's current longitude and latitude.
 You will see a prompt to allow or block this site from knowing your current location. The challenge can be completed either way, as long as the code is correct.
 By selecting allow, you will see the text on the output phone change to your latitude and longitude.
 Here's code that does this:
-<blockquote>if (navigator.geolocation){<br>&nbsp;&nbsp;navigator.geolocation.getCurrentPosition(function(position) {<br>&nbsp;&nbsp;&nbsp;&nbsp;document.getElementById('data').innerHTML="latitude: "+ position.coords.latitude +  "&lt;br&gt;longitude: " +  position.coords.longitude;<br>&nbsp;&nbsp;});<br>}</blockquote>
+
+```js
+if (navigator.geolocation){
+  navigator.geolocation.getCurrentPosition(function(position) {
+    document.getElementById('data').innerHTML="latitude: "+ position.coords.latitude +  "<br>longitude: " +  position.coords.longitude;
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-geolocation-data-to-find-a-users-gps-coordinates.english.md]:
```suggestion
    document.getElementById('data').innerHTML="latitude: " + position.coords.latitude + "<br>longitude: " + position.coords.longitude;
```
Nothing wrong with your conversion, I just figured we might as well make this neater.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Nice work.  Minor suggested changes below.

**Comment by RandellDawson** [comment]
@ojeytonwilliams I have made your suggested changes.

**Review by moT01 — APPROVED** [review]
Went through this again locally - LGTM 🎉

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by RandellDawson** [comment]
@moT01 @ojeytonwilliams See my comment on https://github.com/freeCodeCamp/freeCodeCamp/pull/35993.  I believe this can be merged now.

**Comment by moT01** [comment]
🎉  Nice job with all these everyone 🎉  It will be a nice improvement

**Pull request merged**